zer0-os/zOS
# feat(main): handle and render animated or static background to 'main' with newly added background state

### What does this do?
- handles animated or static background to 'main'

### Why are we making this change?
- to render correct background on main determined by the state.

### How do I test this?
- run tests as usual.
- this is not yet wired up so you won't be able to check in the UI.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
